extensions/amp-mathml/1.0/test/test-component.js

```diff
@@ -0,0 +1,26 @@
+import {expect} from 'chai';
```

Comment:
`expect` is auto defined in our test runner - your ide may have auto imported this but it's safe to remove

Reply:
whoops yeah removed